models/user.ts

```diff
@@ -50,6 +58,10 @@ export const user = createModel<RootModel>()({
       state.avatar = payload.avatar
       return state
     },
+    updateMainPresence: (state, payload: DefaultMainPresence) => {
```

Comment:
if `payload` is a simple object (it's a string) there is no need to declare an extra type (`DefaultMainPresence`), just use `payload: string`